cpp/ps0007/ps0007_solv.cpp

```diff
@@ -36,18 +36,18 @@ int main()
 
   cin>>shape;

```

Comment:
- [x] declared variables to store the user input
- [x] chose suitable data types for the variables
- [x] prompted user to enter data
- [x] used `std::cin` in a right way to get user's input